generators/client-2/templates/src/main/webapp/app/layouts/navbar/_navbar.html

```diff
@@ -165,17 +158,17 @@
                     </ul>
                 </li>
                 <%_ if (enableTranslation){ _%>
-                <li *ngIf="languages" ngbDropdown class="dropdown pointer">
-                    <a class="dropdown-toggle" ngbDropdownToggle href="javascript:void(0);" *ngIf="languages.length > 1">
+                <li *ngIf="languages" ngbDropdown class="nav-item dropdown pointer" >
+                    <a class="nav-link dropdown-toggle" href="javascript:void(0);" id="languagesnavBarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" *ngIf="languages.length > 1">
```

Comment:
data-toggle not required as we use ng-bootstrap from js. bs4 is used only for css